python/cuml/cuml/decomposition/pca.pyx

```diff
@@ -280,11 +280,15 @@ class PCA(UniversalBase,
     _hyperparam_interop_translator = {
         "svd_solver": {
             "arpack": "full",
-            "randomized": "full"
+            "randomized": "full",
+            "covariance_eigh": "full"
```

Comment:
By my reading of the sklearn docs (and our code/docs), I _think_ we could translate both `randomized` and `covariance_eigh` as `jacobi` instead of `full`. Both of these options trade accuracy for performance, which our `jacobi` solver also does IIUC.